substringSearch/console.h

```diff
@@ -0,0 +1,12 @@
+#pragma once
+
+// success: everything is alright;
+// outOfMemory: lack of memory happened
+typedef enum
+{
+    success,
+    outOfMemory = 2
+} ErrorCode;
+
+// Console
+int runProgram(void);
```

Comment:
```suggestion
ErrorCode runProgram(void);
```
Раз уж возвращаем код ошибки, то стоит сказать, что возвращаем код ошибки, а не просто какое-то число